smoketest/scripts/cli/test_protocols_bgp.py

```diff
@@ -87,7 +87,7 @@
         'shutdown'         : '',
         'cap_over'         : '',
         'ttl_security'     : '5',
-        'local_as'         : '300',
+        'system_as'         : '300',
```

Comment:
Please fix indention.

Reply:
I missed this. Thank you for seeing it. It should be fixed now. Sorry :(

Reply:
We will make the adjustments for the Ansible modules in a later stage.